### ISSUE TITLE
[BUG] Using -d flag leads to missing files in output-folder with absolute folder

### ISSUE THREAD

**Issue by tommueller** [compodoc/compodoc#235]
##### **Overview of the issue**

When I use the -d flag to give an output-folder the build of compodoc works fine, but in the output-folder are only the generated files
```
classes
components
coverage.html
graph
index.html
injectables
interfaces
miscellaneous.html
modules
modules.html
overview.html
```
all the static files (js, styles etc.) are missing!

##### **Operating System, Node.js, npm, compodoc version(s)**

Docker node image (7.10) as well as Linux Mint 17 with node 6.11.1 and npm 3.10.10

##### **Compodoc installed globally or locally ?**

Locally

**Comment by vogloblinsky** [comment]
Can i have the command you use ?
And the logs if possible.
Thanks

**Comment by tommueller** [comment]
The command is: `"compodoc-test": "./node_modules/.bin/compodoc src -p tsconfig.json -d /tmp/test"`

And the log is here:
https://pastebin.com/2hCqPRzV

Thanks

**Comment by vogloblinsky** [comment]
Could you test with just `compodoc src -p tsconfig.json -d ./test`

**Comment by vogloblinsky** [comment]
Ok reproduced on my linux machine.

**Comment by tommueller** [comment]
I just realized that it creates a `tmp-folder` with the static files in my project folder. So it's a bug with absolute folder names

**Comment by vogloblinsky** [comment]
Ok fixed, coming in next release 1.0.0-beta.14

**Comment by tommueller** [comment]
Cheers!
